test/new-e2e/runner/local_profile.go

```diff
@@ -26,17 +26,21 @@ func NewLocalProfile() (Profile, error) {
 		return nil, err
 	}
 
+	awsAccount := "agent-sandbox"
 	var store parameters.Store
 	if configPath != "" {
 		configFileValueStore, err := parameters.NewConfigFileValueStore(configPath)
 		if err != nil {
 			return nil, fmt.Errorf("error when reading the config file %v: %v", configPath, err)
 		}
+		if configFileValueStore.Config.ConfigParams.AWS.Account != "" {
```

Comment:
In my opinion this implementation is somewhat hackish. When reading parameters, you're not suppose to know the backing store.

Here your goal is to set environments from a variable. What should be done is to add a parameter in store with `env` that is read and defaulted to `aws/agent-sandbox` if empty.
It should like any param and not limited to the file.

Reply:
Will move default to `store_config_file`, makes sense to move logic there

Reply:
One additional question is if it makes sense to have such a complex setup for local config. Wouldn't it be enough to use the local file ? How many people use env variables ?

Reply:
Side note: ci_profile will require refactoring too, if we want to have a single store in the end and get variables from it.

Reply:
Looking at code, moving this to a store parameter and load it in the common profile code is too risky as `ci_profile` knows about its backing store, plus reads env variables straight from the CI, without the store. I suggest we postpone the refactoring to a follow up PR, let me know if this is blocking for you